pkgs/development/python-modules/fastdotcom/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  icmplib,
+  pythonOlder,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "fastdotcom";
+  version = "0.0.6";
+  format = "setuptools";
+  disabled = pythonOlder "3.8";
+
+  src = fetchPypi {
+    pname = "fastdotcom";
+    inherit version;
+    hash = "sha256-DAj5Bp8Vlg/NQSnz0yF/nHlIO7kStHlBABwvTWHVsIo=";
+  };
+
+  propagatedBuildInputs = [
```

```suggestion
  dependencies = [
```

We use more Python lingo these days.

---

Thanks for the hints. It's been a while since my last Python package contribution.